MuonAnalysis/MomentumScaleCalibration/test/Macros/RooFit/MultiHistoOverlapAll_Z.C

```diff
@@ -331,7 +331,7 @@ void MultiHistoOverlapAll_Z(string files, string labels, string colors = "", str
       }
     }
     leg->Draw("same");
-    cmsPlotTitle->Draw("same");
+    //cmsPlotTitle->Draw("same");
```

Comment:
@usarica 
Just so you're aware, Victor mentioned at the alignment meeting a couple weeks ago (I think you weren't there) that he did not like having CMS preliminary on non-approved plots.  Also there were some plots with other eta cuts so the |eta|<2.4 header did not make sense.  I'm leaving the code here but commenting the Draw()